src/store/modules/recipeStore.js

```diff
@@ -68,6 +68,20 @@ export const recipeStore = {
           context.commit("SET_LOADING_STATE", false);
         });
     },
+
+    async fetch_recipe_raw(context, recipeId) {
+      const uri = this.state.named_urls.recipe_resources + "/" + recipeId;
+      try {
+        const response = await this.state.api.client.get(
+          uri,
+          this.state.api.options
+        );
+        return response.data;
+      } catch (error) {
+        console.error(error);
```

Comment:
There is a context to handle error, use that instead

Reply:
Also the indentation in the entire file is off. Can you fix this please, 4 spaces instead of 2